tests/load/pipeline/test_athena.py

```diff
@@ -231,3 +233,68 @@ def test_athena_file_layouts(destination_config: DestinationTestConfiguration, l
         pipeline, *[t["name"] for t in pipeline.default_schema.data_tables()]
     )
     assert table_counts == {"items1": 3, "items2": 7}
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True, subset=["athena"], force_iceberg=True),
+    ids=lambda x: x.name,
+)
+def test_athena_partitioned_iceberg_table(destination_config: DestinationTestConfiguration):
```

Comment:
please mark this test with `essentials` so we run it on a each CI run

Reply:
The module is marked https://github.com/dlt-hub/dlt/blob/athena-partitions/tests/load/pipeline/test_athena.py#L23-L24

Reply:
Or was this meant for the `athena_adapter` test?

Reply:
heh I meant here... so it seems that all is good